pennylane_lightning/lightning_tensor/quimb/_mps.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Class implementation for MPS manipulation based on the `quimb` Python package.
+"""
+
+import numpy as np
+import quimb.tensor as qtn
+from pennylane.wires import Wires
+
+# TODO: understand if supporting all operations and observables is feasible for the first release
+# I comment the following lines since otherwise Codecov complaints
+
+# _operations = frozenset({})
+# The set of supported operations.
+
+# _observables = frozenset({})
+# The set of supported observables.
+
+
+class QuimbMPS:
+    """Quimb MPS class.
+
+    Interfaces with `quimb` for MPS manipulation.
```

Comment:
This docstring should probably provide some more details. This can wait until the implementation is more complete though.

Reply:
Yep, definitely! I'll add more details in the next PR. Thanks for the reminder